guidelines/governance/wg-governance-guidelines.md

```diff
@@ -0,0 +1,44 @@
+# Working Group Governance Guidelines
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL
+      NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and
+      "OPTIONAL" in this document are to be interpreted as described in
+      [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+## Introduction
+
+The purpose of this document is to describe working group (WG) practices and guidelines. WG members MUST adhere to these guidelines where applicable.
+
+### Relationship to SIGs
+
+All WGs MUST define their relationship to SIGs and follow the related SIGs guidelines. Deviation from related SIGs guidelines MUST be explicitly declared with stated rationale.
+
+## Leadership roles
+
+### Execution Lead
```

Comment:
I have a few questions

- Can you explain the purpose of creating a distinction here between technical lead and execution lead?
- Is this similar to the "chair" role?
- Why would the technical lead not set the priorities of the working group?
- Why are there not roles defined in the sig-governance guidelines?

My thought is that possibly there could be an execution lead or liaison between the sigs, but that the sig technical leads should maintain their roles as technical leads for this project. That also has the effect of eliminating confusion over technical leadership in a group that is explicitly stated to be a collaboration between other groups that already have technical leads

Reply:
It's the equivalent of separating out a product owner from a technical lead. The distinction is that the person responsible for the execution of the project may not necessarily be involved on a technical level. It's possible that these are both the same person, but generally should not be.

There are roles defined in sig-governance. See https://github.com/icon-project/community/blob/594c06d3d22adce71d2b3c6d7e5f1864b67fdfc6/guidelines/governance/sig-governance-guidelines.md#leadership-roles

Reply:
I would say that typically a special interest group is responsible for a project, and only in certain cases is a working group is responsible for a project. By your logic, should there not be an equivalent role in the special interest group?

Reply:
And why should the technical lead generally not be the same as the person responsible for the execution of the project? They are both popular ways to run a project.

Reply:
Correct. The "Chair" role in a SIG would be the equivalent, but it is more broad and strategic. These roles can be honed more as this structure gets adopted.

Reply:
I recommend to remove the technical lead role from the working groups. Execution lead can coordinate with technical leads and chairs for the collaborating sigs